### DISCUSSION TITLE
Change the status of a related element.

### DISCUSSION THREAD

**Discussion by AbbeyDesign** [solspace/craft-freeform#327] [Feature Requests]
I have a successful "edit" form that is mapped to a section. Editing works but I would like to be able to also be able to change the "status" to enabled or disabled, or delete the entry altogether via this form. I don't see the option in the mapping section of the control panel.

**Comment by kjmartens** [comment]
Hi @AbbeyDesign, unfortunately this is not currently possible. However, I will move this to the feature requests list for future consideration.

**Reply by AbbeyDesign** [reply]
ok, thanks!
